oat-sa/extension-lti-proctoring
# Feature/tca 560/create lti proctoring message factory

JIRA ticket: https://oat-sa.atlassian.net/browse/TCA-560

**Problem:** In case of proctor action (pause, terminate) or test completion on unproctored LTI delivery executions TAO has to redirect Test Taker to LTI return url with the same LTI log message as in case of proctored LTI delivery execution.

**Solution:** 
- Extract LTI Message generation from controller method to separate LTI message factory
- Register the new LTI message factory in LTI navigation service (if ltiProctoring is installed)

**How to test:** 
- Start unproctored LTI delivery execution
- Launch LTI proctor monitor with the same `context ID` 
- Pause/terminate delivery execution
- Expected result: Test taker must be redirected to LTI return url with correct LTI log message described in related JIRA ticket

Requires:
- [ ] https://github.com/oat-sa/extension-tao-ltideliveryprovider/pull/249
- [x] https://github.com/oat-sa/extension-tao-proctoring/pull/812
